Include/cpython/bytesobject.h

```diff
@@ -2,6 +2,9 @@
 #  error "this header file must not be included directly"
 #endif
 
+/* Caching the hash (ob_shash) saves recalculation of a bytes' hash value.
+   This significantly speeds up dict lookups. */
```

Comment:
Maybe move this comment before ob_shash definition, in the structure.

Reply:
Or just remove it as trivial? We do not have similar note for other cached hashes, and initially this comment was about strings.